browser/ai_chat/BUILD.gn

```diff
@@ -31,15 +31,21 @@ source_set("browser_tests") {
     testonly = true
     defines = [ "HAS_OUT_OF_PROC_TEST_RUNNER" ]
     sources = [
+      "//chrome/browser/renderer_context_menu/render_view_context_menu_browsertest_util.cc",
+      "//chrome/browser/renderer_context_menu/render_view_context_menu_browsertest_util.h",
```

Comment:
They're in //chrome/test:browser_tests and //chrome/test:interactive_ui_tests targets, neither of them make sense to be added as a deps here, so I add these files to this target.